Example/ConsentViewController/ViewController.swift

```diff
@@ -28,18 +31,38 @@ class ViewController: UIViewController {
             print("There's no consent data stored") :
             print("Deleting following consent data: ", spStoredData)
         consentViewController.clearAllData()
+        vendorXAccepted = false
+        updateCustomVendorUI()
     }
 
     @IBAction func onPrivacySettingsTap(_ sender: Any) {
         consentViewController.loadPrivacyManager()
     }
 
+    @IBAction func onAcceptVendorXTap(_ sender: Any) {
+        consentViewController.customConsentTo(vendors: [ViewController.vendorXId], categories: [], legIntCategories: []) { [weak self] consents in
+            self?.vendorXAccepted = consents.acceptedVendors.contains(ViewController.vendorXId)
+            self?.updateCustomVendorUI()
+        }
+    }
+
+    @IBOutlet weak var vendorXStatusLabel: UILabel!
+    
     override func viewDidLoad() {
         super.viewDidLoad()
         consentViewController.loadMessage()
     }
 }
 
+// MARK: CustomConsent
```

Comment:
should we keep this changes on the example app?

Reply:
I'd like to @leodoin although the UI doesn't look good...